src/dotnet/Program.cs

```diff
@@ -198,6 +198,9 @@ private static void InitializeProcess()
             // by default, .NET Core doesn't have all code pages needed for Console apps.
             // see the .NET Core Notes in https://msdn.microsoft.com/en-us/library/system.diagnostics.process(v=vs.110).aspx
             Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+            // Honor UI language customization
+            UILanguageOverride.Setup();
```

Comment:
I am not a big fan of this kind of comment. If you feel it is needed, how about UILanguageOverride.HonorUILanguageCustomization()?